src/tarkov-data-manager/modules/tarkov-dev-data.mjs

```diff
@@ -34,13 +40,21 @@ const tarkovDevData = {
             try {
                 return JSON.parse(fs.readFileSync(cachePath(filename)));
             } catch (error) {
-                return Promise.reject(error);
+                if (error.code !== 'ENOENT') {
+                    return Promise.reject(error);
+                }
             }
         }
         let newJson = await webSocketServer.getJson(jsonName, gameMode);
         if (newJson.elements) {
             newJson = newJson.elements;
         }
+        if (Array.isArray(newJson) && arrayToDictionary.includes(jsonName)) {
+            newJson = newJson.reduce((all, current) => {
+                all[current.id ?? current._id] = current;
+                return all;
+            }, {});
+        }
```

Comment:
wtf is this? 👀

Reply:
For one data source, the json is in a key:value format, for another, it's in an array. This converts it to key:value so that it can be used consistently within the data manager.